recipes/pytorch_revgrad/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "pytorch_revgrad" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pytorch_revgrad-{{ version }}.tar.gz
+  sha256: 9cf097a7d18cbadddeaec9fef74b258d70b6cb8d0c77f524baab18bffc7d7be9
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.5
+    - flit
+    - numpy
+    - pip
+    - pytorch >=1.0.0
```

Comment:
The package imports itself to find the version during build, the deps aren't actually needed.